source/_integrations/stream.markdown

```diff
@@ -72,3 +72,18 @@ If you see this error you can solve it by running the following commands and res
 ```text
 sudo apt-get install -y python-dev pkg-config libavformat-dev libavcodec-dev libavdevice-dev libavutil-dev libswscale-dev libavresample-dev libavfilter-dev
 ```
+
+Users on FFmpeg < 4 may also see an error similar to:
```

Comment:
Should we delete the troubleshooting section above this since it only applies to an older version of Home Assistant?  In my mind, docs should remain up-to-date.

Reply:
Is it for an older version? I honestly don't remember.

Reply:
I was referring to the install log.  It references av==6.1.2 in the log text

Reply:
Oh good point , removed the old troubleshooting steps :)